addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -1855,7 +1855,25 @@ QUnit.test("basic rendering when sending a message to the followers and thread d
     assert.strictEqual(
         document.querySelector('.o_Composer_followers').textContent.replace(/\s+/g, ''),
         "To:Followersofthisdocument",
-        "Composer should display \"To: Followers of this document\" if the thread as no name."
+        "basic rendering when sending a message to the followers and thread doesn't have a name"
+    );
+});
+
+QUnit.test("basic rendering when sending a message to the followers and thread does have a name", async function (assert) {
```

Comment:
Name of test should be clearer. "Basic rendering" is cryptic. It's okay-ish for test of general template of something, so usually the 1st test of a component. But this one is a single assertion test, so should expect name of test be quite similar to assertion comment.